Classes/Issues/Comments/Html/IssueCommentHtmlCell.swift

```diff
@@ -77,6 +79,23 @@ final class IssueCommentHtmlCell: IssueCommentBaseCell, ListBindable, UIWebViewD
         for (var i = 0; i < imgs.length; i++) {
             imgs[i].addEventListener('click', tapAction);
         }
+        function onElementHeightChange(elm, callback) {
+            var lastHeight = elm.offsetHeight, newHeight;
+            (function run() {
+                newHeight = elm.offsetHeight;
+                if(lastHeight != newHeight) {
+                    callback(newHeight);
+                }
+                lastHeight = newHeight;
+                if(elm.onElementHeightChangeTimer) {
+                    clearTimeout(elm.onElementHeightChangeTimer);
+                }
+                elm.onElementHeightChangeTimer = setTimeout(run, 300);
```

Comment:
This hack is taken from [here](https://stackoverflow.com/a/14901150/940936). There are no DOM events to respond to content size changes. Instead we poll every 300ms and see if the height changed. On change, we try to change the `document.location` (url) to `freetime-hgt://300` where 300 is the new height.

Pretty jank, but this seems to be the right way to handle it.

Another blog post idea!